R/ppm.R

```diff
@@ -1,13 +1,14 @@
 #' PPM - Parts per Million
 #'
-#' `ppm` is a small helper function to mulitply *ppm* values with `1e-6`.
+#' `ppm` is a small helper function to mulitply *mz* values with a given *ppm*.
```

Comment:
Alternative formulation?

`ppm` is a small helper function to determine the parts-per-million for a user-provided value and *ppm*.

Reply:
Yes.